tfe/data_source_workspace.go

```diff
@@ -231,6 +236,7 @@ func dataSourceTFEWorkspaceRead(d *schema.ResourceData, meta interface{}) error
 			"branch":             workspace.VCSRepo.Branch,
 			"ingress_submodules": workspace.VCSRepo.IngressSubmodules,
 			"oauth_token_id":     workspace.VCSRepo.OAuthTokenID,
+			"tags_regex":         workspace.VCSRepo.TagsRegex,
```

Comment:
Could we update the [tests](https://github.com/hashicorp/terraform-provider-tfe/blob/main/tfe/data_source_workspace_test.go) for this.